src/main/java/seedu/address/model/person/Address.java

```diff
@@ -34,7 +34,12 @@ public Address(String address) {
      * Returns true if a given string is a valid address.
      */
     public static boolean isValidAddress(String test) {
-        return test.matches(VALIDATION_REGEX);
+        if (test.contains(" l/ ")) {
+            return test.matches(VALIDATION_REGEX);
+        } else {
+            String[] addressWithTag = test.split( " l/ ");
+            return addressWithTag[0].matches(VALIDATION_REGEX);
+        }
```

Comment:
Ideally, I think we should be validating our fields once we've already isolated them from their labels